perf.groovy

```diff
@@ -341,15 +408,18 @@ def static getOSGroup(def os) {
                 }
             }
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("bash ./tests/scripts/perf-prep.sh --throughput")
```

Comment:
I don't think that we want that to come from the RedHat build.  We just do that building of CoreFX to get binaries to feed into the throughput tests.  If we change the commit that we are grabbing from we will need to make sure that we update baselines on those tests.